index.bs

```diff
@@ -3875,6 +3963,24 @@ To <dfn>check if an [=attribution source=] can create [=aggregatable contributio
     |remainingAggregatableBudget|, return false.
 1. Return true.
 
+To <dfn>find matching aggregatable bucket</dfn> given an [=attribution trigger=] |trigger| and an [=attribution source=] |sourceToAttribute|:
+1. [=list/iterate|For each=] [=aggregatable bucket=] |aggregatableBucket| of |trigger|'s [=attribution trigger/aggregatable buckets=]:
+    1. If the result of running [=match an attribution source against filters and negated filters=]
+        with |sourceToAttribute|, |aggregatableBucket|'s [=aggregatable bucket/filters=],
+        |aggregatableBucket|'s [=aggregatable bucket/negated filters=], and
+        |trigger|'s [=attribution trigger/trigger time=] is true:
+        1. Return |aggregatableBucket|'s [=aggregatable bucket/bucket=].
+1. Return null.
+
+To <dfn>check if an [=attribution source=] can create [=aggregatable contributions=] for matched bucket</dfn> given an
+[=aggregatable attribution report=] |report|, an [=attribution source=] |sourceToAttribute|,
+an [=attribution trigger=] |trigger|, and a [=string=] |matchedBucket|, run the following steps:
```

Comment:
The `|trigger|` parameter is not used by this algorithm.

Reply:
Nice catch, done!